modules/gql/gqlFilters.ts

```diff
@@ -0,0 +1,33 @@
+import { QueryFilterNames } from './gql.constants';
+import { PollsQuery, QueryFilterList } from './types';
+
+export const queryFilters: QueryFilterList = {
+  active: () => ({
+    filter: { endDate: { greaterThanOrEqualTo: Math.floor(Date.now() / 1000) } }
+  }),
+  range: (start, end) => ({
+    filter: {
+      and: [{ pollId: { greaterThanOrEqualTo: start } }, { pollId: { lessThanOrEqualTo: end } }]
+    }
+  }),
+  pollId: pollId => ({ filter: { pollId: { equalTo: pollId } } })
+};
+
+export const getQueryFilter = (filterName: QueryFilterNames | undefined, ...args: any): PollsQuery | null => {
+  if (filterName && queryFilters[filterName]) {
+    switch (filterName) {
+      case QueryFilterNames.Active:
+        return queryFilters.active();
+
+      case QueryFilterNames.Range:
+        return queryFilters.range(args[0], args[1]);
+
+      case QueryFilterNames.PollId:
+        return queryFilters.pollId(args[0]);
+
+      default:
+        break;
+    }
+  }
+  return null;
+};
```

Comment:
@adamgoth This is a bit redundant, initially I was trying to get the following to work by using generic types:

```
export const getQueryFilter = (filterName: QueryFilterNames | undefined, ...args: any): PollsQuery | null => {
  return filterName && queryFilters[filterName] ? queryFilters[filterName](...args) : null;
};
```

But without explicitly passing the parameters to each method, I was getting a typescript error complaining that `a spread argument must either have a tuple type or be passed to a rest parameter`.